### ISSUE TITLE
Stacer for Ubuntu 22.04 

### ISSUE THREAD

**Issue by sauravkhakurel** [oguzhaninan/Stacer#503]
When will stacer for the latest ubuntu22.04 lts version be released? While adding the repo, release file could not be found for ubuntu 22.04. Thanks

**Comment by amiga-500** [comment]
Highly unlikely since the author has abandoned this 3 years ago.  You could compile it on your own for 22.04.  It's pretty easy, even tho the author didnt quite give all the details on how to compile.  Some push requests add some modern updates.

**Comment by sauravkhakurel** [comment]
Could you please point me towards the resources which will guide me through the compilation process? Thanks @amiga-500

**Comment by alexmyczko** [comment]
just apt install stacer? it has been part of ubuntu for years?

**Comment by rendyep** [comment]
use snap instead, the latest version (1.1.0) is available there.

**Comment by sauravkhakurel** [comment]
> use snap instead, the latest version (1.1.0) is available there.

Yeah, did that after I could not successfully compile the code. Thanks for the info
